src/main/java/com/basho/riak/client/core/converters/TimeSeriesPBConverter.java

```diff
@@ -203,10 +202,10 @@ else if(columnType == ColumnDescription.ColumnType.SET) // Set
 
     private static ColumnDescription convertPBColumnDescription(RiakKvPB.TsColumnDescription pbColumn)
     {
-        String name = pbColumn.getName().toStringUtf8();
+        final String name = pbColumn.getName().toStringUtf8();
 
-        ColumnDescription.ColumnType type = ColumnDescription.ColumnType.valueOf(pbColumn.getType().getNumber());
-        List<ColumnDescription.ColumnType> complexType =
+        final ColumnDescription.ColumnType type = ColumnDescription.ColumnType.valueOf(pbColumn.getType().getNumber());
+        final List<ColumnDescription.ColumnType> complexType =
```

Comment:
Complex types was another future proofiing, I'll remove it for now as it's unused, as well as the set/map column type declarations.